src/components/mood-selector/index.tsx

```diff
@@ -68,6 +73,7 @@ const MoodLabel = styled("label", {
       filter: "blur(0)",
       borderColor: "orange",
```

Comment:
this one should be coming from the theme colors I guess?

Reply:
the borderColor: 'orange' - it should be part of the theme and then we should reference it here